src/zap/zapcode.cpp

```diff
@@ -1202,6 +1202,30 @@ void ZapUnwindData::Save(ZapWriter * pZapWriter)
 #endif //REDHAWK
 }
 
+#elif defined(_TARGET_X86_) && defined(FEATURE_PAL)
+
+UINT ZapUnwindData::GetAlignment()
+{
+    return sizeof(BYTE);
```

Comment:
Ok, I was comparing it just to the other platforms. I've missed that comment from @jkotas and he knows better than me the semantics here. So let's ignore all of my comments.